websites/Q/Quran.com/dist/metadata.json

```
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+	  "id": "853932304257908746",
+	  "name": "BigBoi"
+	},
+	"service": "Quran.com",
+	"description": {
+	  "en": "Quran.com is an open source, un-biased, and non-sectarian source of Quranic study that uncovers the beauty of the Quran for Muslims and Non-Muslims. Quran.com offers accessibilty to the Quran translated into many languages in a simple and easy interface."
```

Unless you tell me where you got this from, I prefer using the official one
```suggestion
	  "en": "The Quran translated into many languages in a simple and easy interface"
```

---

Can't remember where I found it but ik for a fact it came from them. Will switch to their default bio for now.